src/_scss/pages/dashboard/graph/_warningsInfoGraph.scss

```diff
@@ -1,5 +1,42 @@
 .warnings-info {
     @import './axis';
+
+    .results-table-content {
+        .title, .description, .loading-message {
+            font-size: rem(30);
+        }
+        .results-table-error {
+            .icon {
+                color: #d9534f;
```

Comment:
Looks like this is redundant since the icon color is already specified [here](https://github.com/fedspendingtransparency/data-act-broker-web-app/pull/1197/files#diff-fc06e1c76c0fc38deaac6824414b2f95R119)